Bottom: 62f4d4743f20ccb238b49247fc21798a98005f55
Top:    0a1761f253ea12742618c66821075a1511a9eb28
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 10:11:25 +0800

Refresh of autofs-5.0.7-fix-bad-mkdir-permission-on-create.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index a22de2d..ef76467 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -124,7 +124,7 @@ static int do_mkdir(const char *parent, const char *path, mode_t mode)
 	    contained_in_local_fs(path)) {
 		mode_t mask = umask(0022);
 		int ret = mkdir(path, mode);
-		void umask(mask);
+		(void) umask(mask);
 		if (ret == -1) {
 			errno = EACCES;
 			return 0;
